### DISCUSSION TITLE
RPC return success but I can not find the transaction in APTOS explorer using response's hash

### DISCUSSION THREAD

**Discussion by OstNo1** [aptos-labs/aptos-developer-discussions#396] [Questions]
### Describe your question in detail.

I sent a signed transaction through RPC https://api.mainnet.aptoslabs.com/v1/transactions, and it responsed 202 with the full transaction detail body, but I can not seach the transaction in APTOS explorer using response's hash, why this happened and what can I do for this? 

### What error, if any, are you getting?

send a signed transaction and RPC response 202 but can not find the transaction in APTOS explorer using response's hash 

### What have you tried or looked at?  Or how can we reproduce the error?

I can provide the transaction data and it reproduces.

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

REST API

**Comment by WGB5445** [comment]
If it's not possible to find this transaction in the browser, I tend to think that the transaction was not sent successfully.

Could you display the details returned by the interface? This would help in better troubleshooting the issue.

**Reply by OstNo1** [reply]
Thanks for reaching out. Here is the signed transaction JSON data:
`{
    "sender": "0x94912706e00bad537c52f8f65026f4ee6b359c03f78cc666b4c3e712f033e1a0",
    "sequence_number": "83",
    "max_gas_amount": "100000",
    "gas_unit_price": "100",
    "expiration_timestamp_secs": "1725360396",
    "payload": {
        "type": "entry_function_payload",
        "function": "0x4bf51972879e3b95c4781a5cdcb9e1ee24ef483e7d22f2d903626f126df62bd1::router::add_liquidity_and_stake_coin_entry",
        "type_arguments": [
            "0x1::aptos_coin::AptosCoin"
        ],
        "arguments": [
            "0x2ebb2ccac5e027a87fa0e2e5f656a3a4238d6a48d93ec9b610d570fc0aa0df12",
            false,
            "7980542",
            "1199999930"
        ]
    },
    "signature": {
        "type": "ed25519_signature",
        "public_key": "102daab444e69b26f52d1ea5276f786f4840ad81a1cc996f3a7c0d979ce4fb06",
        "signature": "eb1197d05d24bf6c2f4daf18daa449f1e9828bdc15383db275e3f636a1e19d8a08af368d2500c3d62d897d5bfd554cb43b9eea777ef425b4b38ab5f6a46f1902"
    }
}`

Here is the response: 
`{
    "hash": "0x155944f7d319cac5dc2546737d288870dd38972888460fe744bb483b9f23eea0",
    "sender": "0x94912706e00bad537c52f8f65026f4ee6b359c03f78cc666b4c3e712f033e1a0",
    "sequence_number": "83",
    "max_gas_amount": "100000",
    "gas_unit_price": "100",
    "expiration_timestamp_secs": "1725360396",
    "payload": {
        "function": "0x4bf51972879e3b95c4781a5cdcb9e1ee24ef483e7d22f2d903626f126df62bd1::router::add_liquidity_and_stake_coin_entry",
        "type_arguments": [
            "0x1::aptos_coin::AptosCoin"
        ],
        "arguments": [
            {
                "inner": "0x2ebb2ccac5e027a87fa0e2e5f656a3a4238d6a48d93ec9b610d570fc0aa0df12"
            },
            false,
            "7980542",
            "1199999930"
        ],
        "type": "entry_function_payload"
    },
    "signature": {
        "public_key": "0x102daab444e69b26f52d1ea5276f786f4840ad81a1cc996f3a7c0d979ce4fb06",
        "signature": "0xeb1197d05d24bf6c2f4daf18daa449f1e9828bdc15383db275e3f636a1e19d8a08af368d2500c3d62d897d5bfd554cb43b9eea777ef425b4b38ab5f6a46f1902",
        "type": "ed25519_signature"
    }
}`

**Reply by WGB5445** [reply]
I see that this response seems normal. Have you tried simulated transactions? Did the simulations succeed? Also, which SDK was used to initiate the transactions?

**Reply by OstNo1** [reply]
In fact, I simulated the transaction and the response in the **arguments** field is  **[ { "inner": "0x2ebb2ccac5e027a87fa0e2e5f656a3a4238d6a48d93ec9b610d570fc0aa0df12" }, false, "7980542", "1199999930" ]**, but using this arguments to send transaction in json format will response error: **Failed to create SignedTransaction from SubmitTransactionRequest: Failed to parse transaction payload: parse arguments[0] failed, expect string<move_struct_tag_id>, caused by error: invalid type: map, expected a string** from the rpc, so I change the the argumets to **[ "0x2ebb2ccac5e027a87fa0e2e5f656a3a4238d6a48d93ec9b610d570fc0aa0df12", false, "7980542", "1199999930" ]**. It worked find when sending to RPC, but can not find the transaction in the browser. 

I used the RPC API  https://api.mainnet.aptoslabs.com/v1/transactions POST  with the JSON format body data metioned above to initiate, NO sdk was used.

**Reply by OstNo1** [reply]
@WGB5445 I can also provide another signed transaction JSON data so that you can reproduce this if needy.
